### PR TITLE
[LILA-7729] README updates

### PR DESCRIPTION
**Pull request by 0xmichalis**
Fix a few typos, broken links, and update
a few pool token references to be agnostic
to a specific pool.

### CODE REVIEW

File: docs/modules.md [hunk 1]
```diff
@@ -27,7 +28,7 @@
 
 #### Defined in
 
-[types/responses.ts:147](https://github.com/GigaHierz/toucan-sdk/blob/b837065/src/types/responses.ts#L147)
+[types/responses.ts:155](https://github.com/0xmichalis/toucan-sdk/blob/1488e66/src/types/responses.ts#L155)
```
Comment (aspiers) [inline docs/modules.md:31]: Why not `ToucanProtocol/toucan-sdk`?

**Review by aspiers — APPROVED** [review]
Great, thanks.  No merge blockers here.  I'll probably do some follow-up work on this too.

File: docs/modules.md (hunk 1 above)
Comment (0xmichalis) [reply docs/modules.md:31]: uhh, this is how `yarn docs` works apparently. I'll raise a ticket to address this as a follow-up.

Comment (0xmichalis) [reply docs/modules.md:31]: Tracking in https://linear.app/toucan/issue/LILA-7809

**Pull request merged**